doc/display.xml

```diff
@@ -98,9 +99,40 @@ gap> Splash(DotDigraph(RandomDigraph(4)));
   </ManSection>
 <#/GAPDoc>
 
+<#GAPDoc Label="DIGRAPHS_DotDigraph">
```

Comment:
We don't usually document functions starting `DIGRAPHS_` these are treated as (kind of) private internal functions/methods.